src/main/rx/rx.c

```diff
@@ -539,6 +541,52 @@ static void detectAndApplySignalLossBehaviour(void)
 #endif
 }
 
+#define MAX_RC_HISTORY_ITEMS 4
+
+uint16_t rcHistory[MAX_SUPPORTED_RC_CHANNEL_COUNT][MAX_RC_HISTORY_ITEMS];
+
+void updateChannelStability(void)
+{
+    for (uint8_t channelIndex = 0; channelIndex < MAX_SUPPORTED_RC_CHANNEL_COUNT; channelIndex++) {
+
+        // calculate the maximum absolute delta between the new reading each each of the previous readings.
+        uint16_t maximumDelta = 0;
+        for (int8_t historyIndex = -1; historyIndex < MAX_RC_HISTORY_ITEMS - 1; historyIndex++) {
```

Comment:
Maybe it would be better to compute min/max over history and use max-min as stability.

This code only compares neighbors, so it is limiting maximum speed .. is that intentional? Is stick stable when moving?